partiql-plan/src/main/resources/partiql_plan.ion

```diff
@@ -322,7 +342,7 @@ rel::{
       _: [
         call::{
           agg:  ref,
-          set_quantifier: [ ALL, DISTINCT ],
+          setq: set_quantifier,
```

Comment:
(self-review) moved `set_quantifier` out of this nested aggregation plan node. Also renamed to `setq` for consistency w/ the set ops.